spt/OrangeBox/modules/ClientDLL.cpp

```diff
@@ -658,16 +663,23 @@ Vector ClientDLL::GetPlayerEyePos()
 
 int ClientDLL::GetPlayerFlags()
 {
+	if (!ORIG_GetLocalPlayer)
+		return 0;
 	return (*reinterpret_cast<int*>(reinterpret_cast<uintptr_t>(ORIG_GetLocalPlayer()) + offFlags));
 }
 
 bool ClientDLL::GetFlagsDucking()
 {
+	if (!ORIG_GetLocalPlayer)
+		return false;
 	return (*reinterpret_cast<int*>(reinterpret_cast<uintptr_t>(ORIG_GetLocalPlayer()) + offFlags)) & FL_DUCKING;
 }
 
 double ClientDLL::GetDuckJumpTime()
 {
+	if (!ORIG_GetLocalPlayer)
+		return 0;
```

Comment:
Why use both `0` and `false`?

Reply:
Don't understand the question, if you can't access the local player data, then no valid value can be returned, in which case it returns 0/false. This is just a hotfix for crashes that are caused by calling a function pointer that is null.

Reply:
Huh? This function returns `double` so it's `0`, the other one returns `bool` so it's `false`.